### PR TITLE
Experience date drop only

### PR DESCRIPTION
**Pull request by eballantine**
What it says on the tin. Plus updating references to 'inventory' to user friendly 'stock'

### CODE REVIEW

File: docs/reference/objects/product/index.md [hunk 1]
```diff
@@ -83,6 +83,13 @@ string
 
 The duration of the product including duration unit e.g. `5 nights`.
 
+## `product.experience_dates`
+{: .d-inline-block }
+[ExperienceDate]({% link docs/reference/objects/product/experience_date.md %})
+{: .label .fs-1 }
+
+An array of the product's [experience dates]({% link docs/reference/objects/product/experience_date.md %})
```
Comment (geoffreymm) [inline docs/reference/objects/product/index.md:91]: Nit: can we add a `.` at the end?

File: docs/reference/objects/product/index.md [hunk 2]
```diff
@@ -83,6 +83,13 @@ string
 
 The duration of the product including duration unit e.g. `5 nights`.
 
+## `product.experience_dates`
+{: .d-inline-block }
+[ExperienceDate]({% link docs/reference/objects/product/experience_date.md %})
```
Comment (geoffreymm) [inline docs/reference/objects/product/index.md:88]:
```suggestion
[Experience Date]({% link docs/reference/objects/product/experience_date.md %})
```

File: docs/reference/objects/product/index.md (hunk 1 above)
Comment (geoffreymm) [reply docs/reference/objects/product/index.md:91]: Should we reference that this is only works when the `type` is experience? Or something like that?

File: docs/reference/objects/product/experience_date.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+## `experience_date.end_on`
+{: .d-inline-block }
+timestamp
+{: .label .fs-1 }
+
+This can then be used in conjunction with Liquid's [built-in filters](https://shopify.github.io/liquid/filters/date/).
+
+## `experience_date.extras`
+{: .d-inline-block }
+array of [Extra]({% link docs/reference/objects/product/extra.md %})s
```
Comment (geoffreymm) [inline docs/reference/objects/product/experience_date.md:24]: Missing `.`

**Review by geoffreymm — APPROVED** [review]
Nice one! 

I think we are just missing:
- id

Given we expose it to and it's necessary for the cart's on sites. besides that looking great!

**Comment by eballantine** [comment]
[Documentation for ExperienceDateDrop](https://trello.com/c/N0rewtq7/147-documentation-for-experiencedatedrop)

**Pull request merged**